staging/src/k8s.io/apiserver/pkg/audit/request.go

```diff
@@ -153,9 +153,13 @@ func LogRequestObject(ae *auditinternal.Event, obj runtime.Object, gvr schema.Gr
 		return
 	}
 
+	copy := copyWithoutManagedFields(obj)
+	if copy == nil {
+		copy = obj
```

Comment:
this is not a good pattern. If you have a func that cannot do its work, either have an error as return type, or at least a `ok` bool.